components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2132,6 +2136,32 @@ void RewardsServiceImpl::ShowNotificationAddFunds(bool sufficient) {
       "rewards_notification_insufficient_funds");
 }
 
+void RewardsServiceImpl::MaybeShowFirstLaunchNotification() {
+  brave_ads::AdsService* ads_service_ =
+    brave_ads::AdsServiceFactory::GetForProfile(profile_);
+
+  if (!ads_service_->ShouldShowAdsNotification()) {
+    return;
+  }
+
+  uint64_t now = std::time(nullptr);
```

Comment:
never use std::time. Always use base::Time

Reply:
but I don't think this is really needed anyway, we should just be creating a timer